cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -260,23 +260,75 @@ export class MigrationWave {
         cy.get("button").contains(stakeHolderGroupName).click();
     }
 
+    private formatDate(date: Date): string {
```

Comment:
This method can be static as we aren't using `this` inside it.

I'd also change the name to `formatDateMMddYYYY` to be clear about what the method actually  does